lib/imgix/path.rb

```diff
@@ -128,14 +128,15 @@ def has_query?
       query.length > 0
     end
 
-    def build_srcset_pairs(widths:, width_tolerance:, params:)
+    def build_srcset_pairs(widths:, width_tolerance:, min_srcset:, max_srcset:, params:)
       srcset = ''
 
       if !widths.empty?
         validate_widths!(widths)
         srcset_widths = widths
-      elsif width_tolerance != DEFAULT_WIDTH_TOLERANCE
-        srcset_widths = TARGET_WIDTHS.call(width_tolerance)
+      elsif width_tolerance != DEFAULT_WIDTH_TOLERANCE or min_srcset != MIN_WIDTH or max_srcset != MAX_WIDTH
+        validate_range!(min_srcset,max_srcset)        
```

Comment:
If you pass in `width_tolerance` but no min or max, won't we try to validate two empty values here and throw an error?

Reply:
`min_srcset` and `max_srcset` will be initialized with default values unless   a user specifies them specifically (see L87).